topics/xml/index.md

```diff
@@ -0,0 +1,11 @@
+---
+created_by: World Wide Web Consortium
+display_name: XML
+released: 1996
+short_description: XML is subset of SGML(standard general markup language) used to create web page. 
```

Comment:
Space after `SGML`. I wonder if it's worth describing what it's used for normally (which is not related to web pages) like your longer description?